### ISSUE TITLE
Flutterflow not getting data from database...

### ISSUE THREAD

**Issue by Dentiphoto** [FlutterFlow/flutterflow-issues#764]
### Has your issue been reported?

- [X] I have searched the existing issues and confirm it has not been reported.
- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

I have an app for ios and android. It has been published. The database on Firebase worked well.

But I had an error deploying push notifications. There has always been a mistake. Yesterday, when I tried again to deploy the push notifications, all the given databases from FlutterFlow were deleted and the connection with Firebase was lost.

I manually recreated all collections and data in FlutterFlow. There are no errors in the project.

I connected the database to Firebase, but in app and run mode there is no data from Firebase.

At the same time, in the application, I create a test user account and it appears in Firebase.

I also got a bug in Firebase functions yesterday. It was written "Waiting for the first deployment".

I do not know what to do. My app is not working.

### Expected Behavior

The data from Firebase should appear in the Application and FlutterFlow Manager.

### Steps to Reproduce

Run mode-only notification:
Firestore Security Rules Error on ContainerMain: Missing or insufficient permissions.

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code

ITEelM/lw4ZIpbxb0M/LYexV9AMwJ0QlUIInj+0bWRMhfLmrPLAEOvT4Q1VLYtTnTVA7BVaYnk8xp/fFkvDlDO1fOgesbLpB+K5XQQLgLVuQRK6LEKuOXER7M99Yf3aayJ7YgAkmPvJvLEYO1zite/CdG3qCf9qOYwx5e6fDbOY=

### Context

Database not working...

### Visual documentation

<img width="1728" alt="Снимок экрана 2023-04-16 в 11 38 53" src="https://user-images.githubusercontent.com/125450278/232287103-e2619024-b859-4a6b-83c4-1036fe730af8.png">
<img width="1728" alt="Снимок экрана 2023-04-16 в 11 38 04" src="https://user-images.githubusercontent.com/125450278/232287108-5a0fc87a-51e8-43c6-8dd0-2005fb418fb1.png">
<img width="1728" alt="Снимок экрана 2023-04-16 в 11 37 54" src="https://user-images.githubusercontent.com/125450278/232287113-44776641-d41b-4c05-9654-e6d2efd7d598.png">
<img width="1728" alt="Снимок экрана 2023-04-16 в 11 37 50" src="https://user-images.githubusercontent.com/125450278/232287119-1a77dfda-b2e7-4770-830f-5b30b4438073.png">
<img width="1728" alt="Снимок экрана 2023-04-16 в 11 37 42" src="https://user-images.githubusercontent.com/125450278/232287126-61e6a8ae-3444-492d-869f-3ba014a31b16.png">
<img width="1321" alt="Снимок экрана 2023-04-16 в 11 26 38" src="https://user-images.githubusercontent.com/125450278/232287130-474a3701-1eea-4fef-8487-ad618c8407f6.png">
<img width="993" alt="Снимок экрана 2023-04-16 в 11 25 45" src="https://user-images.githubusercontent.com/125450278/232287135-83cd2d67-38f3-439b-8baa-50c679e19926.png">
<img width="1725" alt="Снимок экрана 2023-04-16 в 11 21 41" src="https://user-images.githubusercontent.com/125450278/232287138-6067e9a4-662c-4a50-be17-2776c0432779.png">
<img width="1728" alt="Снимок экрана 2023-04-16 в 11 39 02" src="https://user-images.githubusercontent.com/125450278/232287147-f273d1fa-7a86-4110-af99-15d0f74d49a2.png">
<img width="1728" alt="Снимок экрана 2023-04-16 в 11 39 48" src="https://user-images.githubusercontent.com/125450278/232287155-df7024df-e87d-489f-8bc5-6d2e25fe9245.png">

Run mode 
<img width="505" alt="Снимок экрана 2023-04-16 в 11 44 20" src="https://user-images.githubusercontent.com/125450278/232287406-f031fe27-1fd8-4fe0-acbe-283d2ddb46d5.png">

### Environment

```markdown
- FlutterFlow version:
```

**Comment by Dentiphoto** [comment]
<img width="1705" alt="Снимок экрана 2023-04-16 в 11 52 08" src="https://user-images.githubusercontent.com/125450278/232287821-f96b78d7-fa59-40ba-89f7-32fbe8e98107.png">

Flutterflow Content Manager not loading data...

**Comment by Dentiphoto** [comment]
<img width="1728" alt="Снимок экрана 2023-04-16 в 11 54 24" src="https://user-images.githubusercontent.com/125450278/232287906-7d1d20de-d47e-43e0-9a9f-5f78e54c6f63.png">

Nothing in functions...

**Comment by Dentiphoto** [comment]
<img width="1181" alt="Снимок экрана 2023-04-16 в 11 55 52" src="https://user-images.githubusercontent.com/125450278/232287996-aa1553a1-797f-4eb7-99cd-914015f8bb2c.png">

I have a Firefoo data management application and the data from the database is read into the application.

But the data is not being transferred properly to FlutterFlow.

**Comment by hariprasadms** [comment]
Hi @Dentiphoto - Thanks for reporting the issue with details. Looking at first few images, I see a difference in FlutterFlow editor version displayed. Below is the version displaying in my FF editor home page. Not sure why this version difference in your editor. I suggest, do a hard-refresh for your browser ( Ctrl+R) and see the version changes to below version. If any changes, try fire-store rules deployment and see if there are any differences. 

<img width="395" alt="image" src="https://user-images.githubusercontent.com/17435678/232344233-8424df98-de37-40c8-bb26-e1e817a6d90d.png">

Regarding firestore security rules warning message displayed, this happens when security rules are not deployed properly. I suggest, copy rules from FF editor to firebase fire-store rules and save them. Try to access data in app UI and firestore cms manager and see if there are any differences. 

Try these suggestions and let us know how it goes. Also update **Environment** in the issue

**Comment by BirgitPohl** [comment]
I have a similar problem, only that I didn't play around with notifications yet.
I'm just building my app. I have a user's account and a little bit of static content.

What has happened is that after someone in my team re-created the database rules on Firebase directly, I had difficulties re-creating the rules from Flutterflow. So I completely re-connected the Flutterflow with the Firebase instance.
Since then Flutterflow has shown no data whatsoever. It is as if both are out of sync.

I find it strange because I believe there are people who are also connecting existing data from Firebase to Flutterflow. I have seen some Community entries in recent times. So, I expect Flutterflow to handle a back-sync from Firebase.

I have the recent version loaded mentioned above.

What I see as a risk: Building the app and deploying the app to users: Once there is an issue with the connection of Firebase and Flutterflow, we would be forced to recreate the entire database including the data.

**Comment by BirgitPohl** [comment]
So, I figured in my case, that Flutterflow requires read access the same way as any other app, to be able to display data in the Flutterflow view.

I re-created a users table and tried to re-create the same field. I came to the same issue that the users table had an endless loading, but only because Firebase data storage didn't have access rights for it to read. So I set it to true for everyone and then I was able to see the data that was already there.
